build/_k-standard-goals.shade

```diff
@@ -187,6 +189,18 @@ default Quiet='${ false }'
     }
   }
 
+#nuget-verify target='test' if='File.Exists(PACKAGELIST_JSON_FILENAME)' description='Verify if all the packages are generated properly'
+  exec program='cmd' commandline='/C dnu commands install --source ${DNX_TOOLS_FEED} NugetPackageVerifier --packages ${BASE_DIR}\commands\' if='ShouldVerifyNupkgs'
+  exec program='cmd' commandline='/C ${BASE_DIR}/commands/nugetverify ${BUILD_DIR} ${BASE_DIR}\${PACKAGELIST_JSON_FILENAME}' if='ShouldVerifyNupkgs'
+  @{
+    var directory = Path.Combine(BASE_DIR, "commands");
```

Comment:
It isn't *really* repeated, it's just the same value. There's nothing to "refactor" here. This code is perfectly readable and hiding things behind consts will harm that...